cmd/system-probe/config/adjust.go

```diff
@@ -21,7 +21,7 @@ var adjustMtx sync.Mutex
 func Adjust(cfg config.Config) {
 	adjustMtx.Lock()
 	defer adjustMtx.Unlock()
-	if cfg.GetBool(spNS("adjusted")) {
+	if cfg.IsKnown(spNS("adjusted")) {
```

Comment:
Let me known if this change works for you, I could just `SetKnown("adjusted")` too 🤷‍♂️